client/components/feature-modules/entity/components/tables/entity-draft-row.tsx

```
@@ -0,0 +1,146 @@
+        return map;
+    }, [row]);
+
+    // Build ordered cells based on entityType.columns
+    const orderedCells = useMemo(() => {
+        const columnCount = entityType.columns ? entityType.columns.length : 0;
+        if (columnCount === 0) return [];
+
+        // Create maps for quick lookup
+        const attributeCellsMap = new Map(
+            Object.entries(entityType.schema.properties || {}).map(([attributeId, schema]) => {
+                return [attributeId, <EntityFieldCell attributeId={attributeId} schema={schema} />];
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)